client/js/libs/handlebars/ircmessageparser/merge.js

```diff
@@ -0,0 +1,60 @@
+"use strict";
+
+const anyIntersection = require("./anyIntersection");
+const fill = require("./fill");
+
+let Object_assign = Object.assign;
+
+if (typeof Object.assign !== "function") {
```

Comment:
What do you mean? This is the same check than in https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign#Polyfill. I thought you added it but maybe @xPaw did. I didn't touch that part (well I did then I removed lol).